tests/cases/compiler/declarationEmitPrivatePromiseLikeInterface.ts

```diff
@@ -0,0 +1,34 @@
+// @declaration: true
+// @skipLibCheck: true
+// @noTypesAndSymbols: true

```

Comment:
And I've skipped the type/symbol baselines for this test, since they double again the test runtime (and also thus cause a timeout) by _essentially_ invoking the same logic as declaration emit for no real additional gain (compared to the declaration emit we're already running).